### DISCUSSION TITLE
Synthetic Pole Figure Data

### DISCUSSION THREAD

**Discussion by Jinhry** [mtex-toolbox/mtex#1617] [Ask Anything]
Hello there!

I am trying to compare Neutron Diffraction experimental data reliability showing strong fiber texture along the ( 1 1 0 ) direction and I would like to create diverse pole figure data showing similar Fiber texture with varying volume fractions of fiber to vary the texture index (F^2). Is there a way to produce synthetic pole figure data with given input? Thanks for your help!

Sincerely;

**Comment by kilir** [comment]
Hi,
have a look in the documentation.
https://mtex-toolbox.github.io/ODFModeling.html
https://mtex-toolbox.github.io/fibreODF.html

Hope that helps.
Cheets,
Rüdiger
